pypeit/spectrographs/not_alfosc.py

```diff
@@ -195,8 +199,11 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
             exposures in ``fitstbl`` that are ``ftype`` type frames.
         """
         good_exp = framematch.check_frame_exptime(fitstbl['exptime'], exprng)
-        if ftype in ['science', 'standard']:
+        if ftype == 'science':
             return good_exp & (fitstbl['idname'] == 'OBJECT')
+        if ftype in 'standard':
```

Comment:
Should this be `is` or `==`?

Reply:
i would do it `ftype in ['standard']` to maintain flexibility/generality, but `==` is what you want if you want string equality.  iirc, `in` maps the `==` operator across an iterable...